obal/__init__.py

```diff
@@ -237,8 +237,8 @@ def find_targets(inventory_path):
     """
     targets = None
     if os.path.exists(inventory_path):
-        from ansible.inventory.manager import InventoryManager
-        from ansible.parsing.dataloader import DataLoader
+        from ansible.inventory.manager import InventoryManager # pylint: disable=import-outside-toplevel
```

Comment:
great, now this fails on py2, because there the pylint is too old and doesn't know this error.